applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_embedded_solver.py

```diff
@@ -86,19 +94,27 @@ def __init__(self, model, custom_settings):
         if (self.settings["solver_type"].GetString() == "EmbeddedDevelopment"):
             self.element_name = "EmbeddedSymbolicNavierStokes"
 
+        # TODO: Remove this once we finish the new implementations
```

Comment:
does the trilinos version work with the fm_ale?

Reply:
We need to further enhance it to make it compatible (MPI search utility and probably add some calls to the communicator). We have plans to do it soon.